includes/class-wc-payments-order-success-page.php

```diff
@@ -28,39 +28,79 @@ public function __construct() {
 	 */
 	public function register_payment_method_override() {
 		// Override the payment method title on the order received page.
-		add_filter( 'woocommerce_order_get_payment_method_title', [ $this, 'show_woopay_payment_method_name' ], 10, 2 );
+		add_filter( 'woocommerce_order_get_payment_method_title', [ $this, 'show_woocommerce_payments_payment_method_name' ], 10, 2 );
 	}
 
 	/**
 	 * Remove the hook to override the payment method name on the order received page before the order summary.
 	 */
 	public function unregister_payment_method_override() {
-		remove_filter( 'woocommerce_order_get_payment_method_title', [ $this, 'show_woopay_payment_method_name' ], 10 );
+		remove_filter( 'woocommerce_order_get_payment_method_title', [ $this, 'show_woocommerce_payments_payment_method_name' ], 10 );
 	}
 
 	/**
-	 * Add the WooPay logo and the last 4 digits of the card used to the payment method name
-	 * on the order received page.
+	 * Hooked into `woocommerce_order_get_payment_method_title` to change the payment method title on the
+	 * order received page for WooPay and BNPL orders.
 	 *
-	 * @param string            $payment_method_title the default payment method title.
-	 * @param WC_Abstract_Order $abstract_order the order being shown.
+	 * @param string $payment_method_title
+	 * @param WC_Abstract_Order $abstract_order
```

Comment:
```suggestion
	 * @param string $payment_method_title Original default payment method title.
	 * @param WC_Abstract_Order $abstract_order Successful received order being shown.
```
Not a huge deal, but it's proper documentation form to include a description of the parameters at the end of the doc comment here.

Reply:
Resolved in 6744f07a7